### PR TITLE
subscriptions: Migrate to webpack

### PR DESCRIPTION
**Pull request by larsu**
This obsoletes #4691, amending @stefwalter's commit. Adds index.html and manifest.json to the webpack config and calls init() after the page is ready (took me way too long to figure this one out :/).

Should I mention in the commit message that I changed the original patch?

### CODE REVIEW

**Comment by dperpeet** [comment]
build fails

**Comment by larsu** [comment]
Ah, the rename didn't make it into `POTFILES.in`. Thanks.

File: pkg/subscriptions/index.html [hunk 1]
```diff
@@ -24,11 +24,9 @@
...
   <link href="../base1/cockpit.css" rel="stylesheet">
   <link href="subscriptions.css" rel="stylesheet">
-  <script type="text/javascript" src="../base1/bundle.js"></script>
-  <script type="text/javascript" src="bundle.js"></script>
-  <!-- debug:start -->
-    <script src="subscriptions.js"></script>
-    <!-- debug:end -->
+  <script type="text/javascript" src="../base1/jquery.js"></script>
+  <script type="text/javascript" src="../base1/cockpit.js"></script>
+  <script type="text/javascript" src="subscriptions.js"></script>
```
Comment (stefwalter) [inline pkg/subscriptions/index.html]: Need to load shell/po.js here.

**Comment by stefwalter** [comment]
Just one change needed, other than that looks good to me.

**Comment by larsu** [comment]
Ah right. Updated.

**Comment by dperpeet** [comment]
let's port to react first: #4715

**Comment by stefwalter** [comment]
> let's port to react first: #4715

Why? #4715 is WIP and massive, and this is a tiny little change.

**Comment by stefwalter** [comment]
This has come up twice on this pull request, it's likely related:

```
Wrote TestSubscriptions-testRegister-FAIL.png
Journal extracted to TestSubscriptions-testRegister-10.111.126.158-FAIL.log
Journal extracted to TestSubscriptions-testRegister-10.111.114.82-FAIL.log
not ok 83 testRegister (check_subscriptions.TestSubscriptions)
Traceback (most recent call last):
  File "./verify/check-subscriptions", line 110, in testRegister
    b.wait_in_text("body", "Invalid login or password.")
  File "/data/src/cockpit/test/common/testlib.py", line 241, in wait_in_text
    return self.wait_js_func('ph_in_text', selector, text)
  File "/data/src/cockpit/test/common/testlib.py", line 211, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "/data/src/cockpit/test/common/testlib.py", line 665, in <lambda>
    return lambda *args: self._invoke(name, *args)
  File "/data/src/cockpit/test/common/testlib.py", line 688, in _invoke
    raise Error(res['error'])
Error: timeout
```

File: pkg/subscriptions/main.js [hunk 2]
```diff
@@ -617,5 +611,5 @@ require([
         subscriptions.manager = subscription_manager(retrieve_current);
     }
 
-    init();
-});
+    $(init);
```
Comment (stefwalter) [inline pkg/subscriptions/main.js]: You need to move register_dialog_initialize() into the init function, or call it once the page has loaded.

Comment (larsu) [inline pkg/subscriptions/main.js]: Done, thanks.

**Comment by larsu** [comment]
> Why? #4715 is WIP and massive, and this is a small change. 

On second thought, I agree. Let's unblock this.

@dperpeet I can rebase your branch once this lands.

**Comment by stefwalter** [comment]
Making sure fedora-24 is tested before merging. Although this seems unrelated, it's a good cross check.

**Pull request closed without merging**